cmd/version.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// versionCmd represents the version command
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Use 'gscloud <version> [e.g. docker]' to return version",
+	Long: `With 'version' you are not only able to see which gscloud version you are running,
+	it is possible to check versions of e.g. 'docker' as well`,
+	Run: func(cmd *cobra.Command, args []string) {
+		// TODO get information through a json file and add more cases
+		if len(args) == 0 {
+			fmt.Println("v0.2.0-beta")
+		}
+		for _, arguments := range args {
+			switch arguments {
+			case "docker":
+				fmt.Println("19.03.8")
```

Comment:
@c0dehard would explain why do you check the docker version here?

Reply:
it was just a test, i renamed it already